src/MoBi.Presentation/Presenter/EditTableFormulaWithOffSetFormulaPresenter.cs

```diff
@@ -42,22 +48,22 @@ public class EditTableFormulaWithOffSetFormulaPresenter : EditTypedFormulaPresen
       public override void Edit(TableFormulaWithOffset tableFormulaWithOffset)
       {
          _formula = tableFormulaWithOffset;
-         _tableFormulaWithOffsetDTO = _mapper.MapFrom(_formula);
-         _view.BindTo(_tableFormulaWithOffsetDTO);
+         bindToView();
       }
 
-      private void rebind()
+      private void bindToView()
       {
-         Edit(_formula);
```

Comment:
This was actually a bug... setting the Owner to null on rebind and then the view would crash. Interesting how no-one has ever used this feature :)